Formula/csound.rb

```diff
@@ -50,14 +57,14 @@ def install
     args = std_cmake_args + %W[
       -DABLETON_LINK_HOME=#{buildpath}/ableton
       -DBUILD_ABLETON_LINK_OPCODES=ON
-      -DBUILD_JAVA_INTERFACE=OFF
```

Comment:
I think it's better to explicitly turn it on

Reply:
This is done (commit b6616646f3400ae215b0356f96298f57cea24b42).